client/src/components/App.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import Reservation from './Reservations'
+const ROOM = 42;
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      availNights: []
+    }
+  }
+  componentDidMount () {
+    this.fetch(ROOM);
+  }
+  fetch (roomId) {
+    axios.get(`/rooms/${roomId}`)
+      .then(db => this.setState({availNights: db.data}))
+      .catch(err => console.error(err));
+  }
+  render() {
+    return (
+      <div>
+        <Reservation />
```

Comment:
Will the availNights state be passed into Reservation as a prop?

Reply:
Yes, at some point. Haven’t gotten there yet. Just checking to see if the data was getting sent to the client.